src/main/resources/application.yml

```diff
@@ -18,6 +18,9 @@ logging:
         web:
           client:
             RestTemplate: DEBUG
+security:
+  token:
+    ttl: 3600
```

Comment:
Making it a Duration will make it more transparent 
https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/Duration.html
Having said that, 3600 seconds can be represented by a bigger unit of time.